packages/embark-coverage/src/lib/instrumenter.ts

```diff
@@ -28,10 +30,17 @@ export class Instrumenter {
       return;
     }
 
-    this.addInjectionPoints("contractDefinition", 1, node.loc);
+    this.addInjectionPoints("contractDefinition", 1, node);
   }
 
   public instrumentFunction(node: FunctionDefinition) {
+    // Remove stateMutability so that the function is not exported
+    // as "view" or "pure", seen as it will contain some emit
+    // statements and those require a transaction.
```

Comment:
💯 this is much better than removing them from the code, very nice!

Reply:
Man, you have no idea how happy I am that you came by. I mostly did things this way because I was trying to impress you.

Reply:
I am definitely impressed

Reply:
We miss u @alaibe !